### PR TITLE
feat: apply read write access at service pages

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### Things to Talk About
I've applied UI adjustments based on the permissions for read-only and write access on each service page. It wasn't difficult since most of the code was repetitive!

### CODE REVIEW

File: apps/web/src/lib/access-control/page-access-helper.ts [hunk 1]
```diff
@@ -48,14 +48,13 @@ export const getPageAccessMapFromRawData = (pageAccessPermissions?: string[]): P
...
-        const access = accessType !== PAGE_ACCESS.READ_ONLY;
+        const access = accessType !== PAGE_ACCESS.NO_ACCESS;
 
-        setPermissions(menuId, read, write, access);
+        setPermissions(menuId, access, write, access);
 
         menu.subMenuList?.forEach(({ id: subMenuId }) => {
-            setPermissions(subMenuId, read, write, access);
+            setPermissions(subMenuId, access, write, access);
         });
```
Comment (piggggggggy) [inline apps/web/src/lib/access-control/page-access-helper.ts:58]: Question: conditions of `read` and `access` are same?

**Review by sulmoJ — APPROVED** [review]
LGTM~

**Review by piggggggggy — APPROVED** [review]
LGTM~~ I left small comment

File: apps/web/src/lib/access-control/page-access-helper.ts (hunk 1 above)
Comment (skdud4659) [reply apps/web/src/lib/access-control/page-access-helper.ts:58]: yup

**Pull request merged**